### DISCUSSION TITLE
Добавьте AHA Music - Song Finder for Browser

### DISCUSSION THREAD

**Discussion by Fikus123123** [anticensority/runet-censorship-bypass#196] [Ideas]
Не загружаются треки с адреса[[ [AHA Music - Song Finder for Browser](https://chrome.google.com/webstore/detail/aha-music-song-finder-for/dpacanjfikmhoddligfbehkpomnbgblf)](https://www.aha-music.com/6473a5ab205a2af83aaaeaa85e79f31a?utm_source=chrome&utm_medium=extension)э
](https://www.aha-music.com/6473a5ab205a2af83aaaeaa85e79f31a?utm_source=chrome&utm_medium=extension)

**Comment by ilyaigpetrov** [comment]
aha-music.com не заблокирован: см. https://reestr.rublacklist.net/ru/?q=aha-music.com.
Возможно, блокируется какой-то другой адрес -- попробуйте вяснить это самостоятельно: см. https://github.com/anticensority/runet-censorship-bypass/wiki/Чиним-сайты-со-сломанными-элементами.
